mtbf_driver/mtbf_apps/contacts/app.py

```diff
@@ -0,0 +1,48 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from marionette.by import By
+from marionette import expected
+from marionette.errors import JavascriptException
+from gaiatest.apps.contacts.app import Contacts
+from marionette import Wait
+from gaiatest.apps.base import Base
+
+
+class MTBF_Contacts(Contacts):
+
+    _edit_form_title_locator = (By.ID, 'contact-form-title')
+    _edit_form_close_locator = (By.ID, 'contact-form-header')
+
+    def launch(self):
+        Base.launch(self)
+
+        Wait(self.marionette, ignored_exceptions=JavascriptException).until(
+            lambda m: m.execute_script('return window.wrappedJSObject.Contacts.asyncScriptsLoaded;') is True)
+
+        self.close_existing_edit_form()
+
+        self.clear_search_view()
+
+        Wait(self.marionette).until(expected.element_displayed(
+            Wait(self.marionette).until(expected.element_present(
+                *self._settings_button_locator))))
+
+    def clear_search_view(self):
+        if self.marionette.find_element(*(By.ID, 'cancel-search')).is_displayed():
```

Comment:
Would you mind storing found element as reference variable and use the shortened name later?